ocf/ocf.go

```diff
@@ -59,6 +59,20 @@ type decoderConfig struct {
 	CodecOptions  codecOptions
 }
 
+func newDecoderConfig(opts ...DecoderFunc) *decoderConfig {
+	cfg := decoderConfig{
+		DecoderConfig: avro.DefaultConfig,
+		SchemaCache:   avro.DefaultSchemaCache,
+		CodecOptions: codecOptions{
+			DeflateCompressionLevel: flate.DefaultCompression,
+		},
+	}
+	for _, opt := range opts {
+		opt(&cfg)
+	}
+	return &cfg
+}
+
```

Comment:
Not sure that this change helped anything.

Reply:
Avoiding duplicate code.